src/parsers/rss_channel.c

```diff
@@ -353,14 +352,12 @@ rss_init_feed_handler (void)
 		g_hash_table_insert (RssToMetadataMapping, "author", "author");
 		g_hash_table_insert (RssToMetadataMapping, "comments", "commentsUri");
 	}
-	
+
 	if (!rss_nstable) {
 		rss_nstable = g_hash_table_new (g_str_hash, g_str_equal);
 		ns_rss_ns_uri_table = g_hash_table_new (g_str_hash, g_str_equal);
 
-		/* register name space handlers */		
-		rss_add_ns_handler (ns_bC_get_handler ());
-		rss_add_ns_handler (ns_dc_get_handler ());
```

Comment:
You removed blogChannel but ns_dc_get_handler() is removed too, was that intended ?

Reply:
Oh... Reverted.